docs/Sources/tuist/tuist.docc/Articles/Tuist/Users/sharing-code-across-manifests.md

```diff
@@ -0,0 +1,85 @@
+
+# Sharing code across manifests
+
+Project description helpers are a group of Swift files that can be accessed from any project manifest. They are useful to extract common patterns, define project conventions, or simplify the definition of projects.
+
+## Overview
+
+One of the inconveniences of Xcode when we use it with large projects is that it doesn't allow reusing elements of the projects other than the build settings through `.xcconfig` files. Being able to reuse project definitions is useful for the following reasons:
+
+- It eases the **maintenance** because changes can be applied in one place and all the projects get the changes automatically.
+- It makes it possible to define **conventions** that new projects can conform to.
+- Projects are more **consistent** and therefore the likelihood of broken builds due inconsistencies is significantly less.
+- Adding a new projects becomes an easy task because we can reuse the existing logic.
+
+Reusing code across manifest files is possible in Tuist thanks to the concept of **project description helpers**.
+
+### Definition
+
+Project description helpers are Swift files that get compiled into a framework, `ProjectDescriptionHelpers`, that manifest files can import.
```

Comment:
I'd say `into a module` instead of `into a framework` so that we can change the implementation details in the future without impacting the documentation.